css/main.css

```diff
@@ -396,6 +399,9 @@ footer {
 
   .logo-img {
     width: 400px;
+    height: 400px;
+    overflow-anchor: none;
```

Comment:
Here too, same comment

Reply:
Yes, these lines are required, if deleted it will disturb the dimensions of the logo like this:
![image](https://user-images.githubusercontent.com/52736997/148384863-65f4a3af-5757-4494-ae6d-863e3b4dd3ba.png)

Reply:
What if we only remove `z-index` ?? Will this be same in that case ??

Reply:
Removing `z-index` from the classes